pages/golfers/404.js

```diff
@@ -0,0 +1,10 @@
+const NotFoundPage = () => {
+    return (
```

Comment:
The spacing is inconsistent throughout the PR and the rest of the code. For instance, here you are indenting with 4 spaces. Meanwhile, in `lib/useScoresById.js` you use 2 spaces. I think that you should use 2 spaces everywhere.